bin/trigger_azure_pipelines.py

```diff
@@ -80,19 +112,30 @@ def main() -> None:
         pipeline_id = os.environ["ADO_AZUREAUTH_LINUX_PIPELINE_ID"]
         version = os.environ["VERSION"]
         commit_hash = os.environ["GITHUB_SHA"]
+        ado_artifact_name = os.environ["ADO_LINUX_ARTIFACT_NAME"]
+        artifact_download_path = os.environ["ADO_LINUX_ARTIFACT_DOWNLOAD_PATH"]

```

Comment:
Is the download path expected to be something that changes or we need to hide? I could see this download path being computed from the artifact name.

Reply:
The path we want to download to during GitHub actions is `dist/linux-x64` because the other platform binaries are created under `dist/runtime`. So did not create it from the artifact name.